frontend/src/components/PageNotFoundComponent.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="container">
+    <v-img src="../assets/404logo.png" id="err-logo"></v-img>
+    <v-img src="../assets/404img.png" alt="logo" id="err-img"></v-img>
+    <div class="row">
+      <v-btn
+        type="ReturnToHome"
```

Comment:
the button type should be `text`, `number`, etc.. not `ReturnToHome, Please update